dkg/dataclasses.py

```diff
@@ -55,8 +55,9 @@ class KnowledgeAssetContentVisibility(AutoStrEnumUpperCase):
     PRIVATE = auto()
 
 
-class ParanetIncentivizationType(AutoStrEnumCapitalize):
-    NEUROWEB = auto()
+class ParanetIncentivizationType(Enum):
+    NEUROWEB = "Neuroweb"
+    ERC20 = "NeurowebERC20"
```

Comment:
Should we call it `NEUROWEB_ERC20`? There is no support for any custom ERC20 atm, so perhaps it would be more clear